install/deploy_management_nodes.md

```diff
@@ -525,9 +537,12 @@ The configuration workflow described here is intended to help understand the exp
     ncn-s# cephadm shell -- ceph-volume inventory --format json-pretty | jq -r '.[]|select(.available==true)|.path'
     ```
 
-1. Wipe the drive ONLY after you have confirmed the drive is not being used by the current Ceph cluster via options 1, 2, or both.
+##### Wipe and Add Drives
+
+1. Wipe the drive `ONLY after you have confirmed the drive is not being used by the current Ceph cluster` via options 1, 2, or both.
```

Comment:
Is the goal to emphasize the text? Should be using bold or italics not monospace.